manage.py

```diff
@@ -254,7 +254,7 @@ def cf_startup():
     """Migrate schemas on `cf push`."""
     check_config()
     if env.index == '0':
-        subprocess.Popen(['python', 'manage.py', 'refresh_materialized'])
+        subprocess.Popen(['python', '', 'refresh_materialized'])
```

Comment:
I don't think you meant to delete this.

Reply:
good catch - I think it is an accident. I rebased manage.py from develop branch. Thx.